doc/features/ffmpeg/README.md

```diff
@@ -0,0 +1,9 @@
+# ffmpeg-qsv capabilities
```

Comment:
Maybe make the title of the landing page more official and to the point.  Something like:
"Codec capabilities of Intel® Quick Sync Video (QSV) with FFmpeg"

Reply:
done, but w/ slightly different title